akka-distributed-data/src/main/mima-filters/2.5.15.backwards.excludes

```diff
@@ -0,0 +1,4 @@
+# #25543 allow BigInt increment and decrements in PNCounter
+ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.cluster.ddata.PNCounter.increment")
+ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.cluster.ddata.PNCounter.decrement")
+ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.cluster.ddata.PNCounter.change")
```

Comment:
do new lines matter in these files?

Reply:
Afair that's only for reference.conf since those could be concatenated together by some super-jar kind of build step in projects (and then no linefeed causes invalid conf).